packages/charts/src/components/zero-intersection.utils.test.ts

```diff
@@ -0,0 +1,139 @@
+import { scaleBand, scaleLinear } from 'd3-scale';
+
+import {
+  getZeroIntersectionVisibility,
+  calculateZeroIntersection,
+} from './zero-intersection.utils';
+
+describe('@keen.io/charts', () => {
```

Comment:
We decided to use `test` instead of `it` in unit tests. Additionally `describe('@keen.io/charts')` is not required as JEST displays each package name in CI.